insights/parsers/pci_rport_target_disk_paths.py

```diff
@@ -0,0 +1,219 @@
+class PCIRportTargetDiskPaths(CommandParser, dict):
+    """
+    Class for parsing ``find /sys/devices/pci0000:00 -mindepth 8 -maxdepth 8`` command output.
+
+    Typical output of command ``find /sys/devices/pci0000:00 -mindepth 8 -maxdepth 8`` with
+    the filter of 'block' looks like::
+
+        /sys/devices/pci0000:00/0000:00:01.0/0000:04:00.6/host1/rport-1:0-1/target1:0:0/1:0:0:0/block/sdb
+        /sys/devices/pci0000:00/0000:00:01.0/0000:04:00.7/host2/rport-2:0-2/target2:0:0/2:0:0:0/block/sdc
+        /sys/devices/pci0000:00/0000:00:02.2/0000:02:00.0/host0/target0:1:0/0:1:0:0/block/sda/dev
+        /sys/devices/pci0000:00/0000:00:02.2/0000:02:00.0/host0/target0:1:0/0:1:0:0/block/sda/sda1
+        /sys/devices/pci0000:00/0000:00:02.2/0000:02:00.0/host0/target0:1:0/0:1:0:0/block/sda/sda2
+
+    The Original data parsed looks like::
+
+            {
+                'PCI': [
+                    {
+                        'target': 'target1: 0: 0',
+                        'devnode': 'sdb',
+                        'host_channel_id_lun': '1: 0: 0: 0',
+                        'pci_id': '0000: 04: 00.6',
+                        'host': 'host1',
+                        'rport': 'rport-1: 0-1'
+                    },
+                    {
+                        'target': 'target2: 0: 0',
+                        'devnode': 'sdc',
+                        'host_channel_id_lun': '2: 0: 0: 0',
+                        'pci_id': '0000: 04: 00.7',
+                        'host': 'host2',
+                        'rport': 'rport-2: 0-2'
+                    },
+                    {
+                        'target': 'target0: 1: 0',
+                        'devnode': 'sda',
+                        'host_channel_id_lun': '0: 1: 0: 0',
+                        'pci_id': '0000: 02: 00.0',
+                        'host': 'host0',
+                        'rport': 'rport-2: 0-2'
+                    }
+                        ]
+            }
+
+    Examples:
+        >>> type(pd)
+        <class 'insights.parsers.pci_rport_target_disk_paths.PCIRportTargetDiskPaths'>
+        >>> pd.pci_id
+        ['0000:02:00.0', '0000:04:00.6', '0000:04:00.7']
+        >>> pd.host
+        ['host0', 'host1', 'host2']
+        >>> pd.target
+        ['target0:1:0', 'target1:0:0', 'target2:0:0']
+        >>> pd.host_channel_id_lun
+        ['0:1:0:0', '1:0:0:0', '2:0:0:0']
+        >>> pd.devnode
+        ['sda', 'sdb', 'sdc']
+
+    Attributes:
+        host (list): The list of all host(s)
+        rport (list): The list of all rport(s)
+        target (list): The list of all target(s)
+        devnode (list): The list of all devnode(s)
+        pci_id (list): The list the all pci_id(s)
+        host_channel_id_lun (list): The list of all host_channel_id_lun(s)
+
+    Raises:
+        ParseException: Input content is not available to parse
+        SkipException: Input content is empty
+
+    """
+
+    def __attrs__(self, item):
```

Comment:
This could just be created as a private attribute with a descriptive name like `__device_attributes` in the `parse_content` method, instead of calling a method every time a property is accessed.

Reply:
@bfahr good suggestion!
after line 205, I created few private attributes with corresponding descriptive names as you suggested, these attributes are invoked by class functions. Does it look ok to you?